azure-storage-cpp-lite/include/retry.h

```diff
@@ -67,8 +68,11 @@ namespace microsoft_azure {
             retry_info evaluate(const retry_context &context) const override {
                 if (context.numbers() == 0) {
                     return retry_info(true, std::chrono::seconds(0));
-                } else if (context.numbers() < 100 && can_retry(context.result())) {
-                    return retry_info(true, std::chrono::seconds(5+rand()%5));
+                } else if (context.numbers() < 60 && can_retry(context.result())) {
+                    double delay = (pow(2, context.numbers()-1)-1);
```

Comment:
I think we should reduce the base here, maybe 1.2?  2^x can grow quite quick, I think it would be good to have a few quick retries before starting to get into longer backoffs.

Reply:
Yes, it caps out at 60 seconds